src/components/TeacherViews/Onboarding/HowCanOakSupport/HowCanOakSupport.view.tsx

```diff
@@ -87,11 +87,14 @@ const HowCanOakSupport = () => {
         trigger={trigger as UseFormTrigger<OnboardingFormProps>}
         forceHideNewsletterSignUp={true}
         subheading="Select all that apply"
-        secondaryButton={
-          <OakSecondaryButton width="100%" disabled={hasMissingFormData}>
+        secondaryButton={(isSubmitting) => (
+          <OakSecondaryButton
+            width="100%"
+            disabled={hasMissingFormData || isSubmitting}
+          >
             Skip
           </OakSecondaryButton>
-        }
+        )}
```

Comment:
It was weird that the "Skip" button wasn't also disabled when the form was submitted. So i've made `secondaryButton` a render prop so that we can conditionally render it depending on the form state.